process/resultsParser/returnCode.go

```diff
@@ -0,0 +1,27 @@
+package resultsParser
+
+import (
+	"bytes"
+	"encoding/hex"
+)
+
+// ReturnCode is an enum that describes some common smart contract results.
+type ReturnCode string
+
+const (
+	None       ReturnCode = ""
+	Ok         ReturnCode = "ok"
```

Comment:
please remove these constans, no longer used.

Reply:
removed the file altogether. thanks for pointing it out.